test/SlowTests/Server/Documents/ETL/EtlTimeSeriesTests.cs

```diff
@@ -1713,9 +1713,10 @@ await using (OpenEtlOffArea(src, etlResult.TaskId, true))
                 var entity = new User { Name = "Joe Doe" };
                 await session.StoreAsync(entity, documentId);
 
+                var ts = session.TimeSeriesFor(documentId, timeSeriesName);
```

Comment:
Can you explain why does it help? Also it should go to 5.2 IMO

Reply:
Yes - every time we call `session.TimeSeriesFor(documentId, timeSeriesName)` we are allocating a new instance of `AsyncSessionDocumentTimeSeries<TimeSeriesEntry>` instead of doing it once. Also, I checked it with & without the fix in **dotMemory** and the memory is lower (fewer instances created) so I think it can fix the test

Reply:
I guess this test fails as a part of this issue:
https://issues.hibernatingrhinos.com/issue/RavenDB-19879/